### PR TITLE
Release Alt-Ergo 2.3.3 (all packages)

### PR DESCRIPTION
**Pull request by OCamlPro-Coquera**

### CODE REVIEW

**Comment by avsm** [comment]
@OCamlPro-Coquera could you please indicate when this is ready for review?

File: packages/alt-ergo-lib/alt-ergo-lib.2.3.3/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+"This is the core library used in the Alt-Ergo SMT solver.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+
+url {
+  src:
+    "http://alt-ergo.ocamlpro.com/http/alt-ergo-2.3.3/alt-ergo-2.3.3.tar.gz"
+  checksum: "md5=776406c13df1abd65797524deff5f66c"
```
Comment (OCamlPro-Coquera) [inline packages/alt-ergo-lib/alt-ergo-lib.2.3.3/opam:34]:
```suggestion
  checksum: "md5=3b060044767d16d1de3416944abd2dd5"
```

File: packages/alt-ergo-parsers/alt-ergo-parsers.2.3.3/opam [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+"This is the parser library used in the Alt-Ergo SMT solver.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+
+url {
+  src:
+    "http://alt-ergo.ocamlpro.com/http/alt-ergo-2.3.3/alt-ergo-2.3.3.tar.gz"
+  checksum: "md5=776406c13df1abd65797524deff5f66c"
```
Comment (OCamlPro-Coquera) [inline packages/alt-ergo-parsers/alt-ergo-parsers.2.3.3/opam:34]:
```suggestion
  checksum: "md5=3b060044767d16d1de3416944abd2dd5"
```

File: packages/alt-ergo/alt-ergo.2.3.3/opam [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+synopsis: "The Alt-Ergo SMT prover"
+description:
+"Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+
+url {
+  src:
+    "http://alt-ergo.ocamlpro.com/http/alt-ergo-2.3.3/alt-ergo-2.3.3.tar.gz"
+  checksum: "md5=776406c13df1abd65797524deff5f66c"
```
Comment (OCamlPro-Coquera) [inline packages/alt-ergo/alt-ergo.2.3.3/opam:29]:
```suggestion
  checksum: "md5=3b060044767d16d1de3416944abd2dd5"
```

File: packages/altgr-ergo/altgr-ergo.2.3.3/opam [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+"Altgr-Ergo is the graphical interface for the Alt-Ergo SMT prover.
+
+Alt-Ergo is an automatic theorem prover of mathematical formulas. It was developed at LRI, and is now maintained at OCamlPro.
+
+See more details on http://alt-ergo.ocamlpro.com/"
+
+url {
+  src:
+    "http://alt-ergo.ocamlpro.com/http/alt-ergo-2.3.3/alt-ergo-2.3.3.tar.gz"
+  checksum: "md5=776406c13df1abd65797524deff5f66c"
```
Comment (OCamlPro-Coquera) [inline packages/altgr-ergo/altgr-ergo.2.3.3/opam:33]:
```suggestion
  checksum: "md5=3b060044767d16d1de3416944abd2dd5"
```

File: packages/alt-ergo-lib/alt-ergo-lib.2.3.3/opam [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+depends: [
+  "ocaml" {>="4.04.0"}
+  "dune" {>= "1.5"}
+  "num"
+  "ocplib-simplex" {>= "0.4" }
+  "zarith"
+  "seq"
+  "stdlib-shims"
+]
+homepage: "http://alt-ergo.ocamlpro.com/"
```
Comment (OCamlPro-Coquera) [inline packages/alt-ergo-lib/alt-ergo-lib.2.3.3/opam:19]:
```suggestion
conflicts: [
  "alt-ergo-free"
  "alt-ergo-lib-free"
]
homepage: "http://alt-ergo.ocamlpro.com/"
```

**Comment by OCamlPro-Coquera** [comment]
@avsm i was waiting for the merge of https://github.com/OCamlPro/alt-ergo/pull/353, this PR should be ready to review.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Comment by OCamlPro-Coquera** [comment]
Thanks

**Pull request merged**